internal/testrunner/runners/system/runner.go

```diff
@@ -692,32 +760,58 @@ func (r *runner) prepareScenario(ctx context.Context, config *testConfig, servic
 		return nil, fmt.Errorf("failed to find the selected policy_template: %w", err)
 	}
 
-	// Setup service.
-	logger.Debug("setting up service...")
-	serviceDeployer, err := servicedeployer.Factory(serviceOptions)
-	if err != nil {
-		return nil, fmt.Errorf("could not create service runner: %w", err)
+	// Configure package (single data stream) via Fleet APIs.
```

Comment:
Move creation of test Agent Policy as one of the first steps, so it can be used to enroll the new agents.